python/py_unzip/__main__.py.tmpl

```diff
@@ -0,0 +1,128 @@
+%shebang%
+
+# Copyright 2019 The Bazel Authors. All rights reserved.
```

Comment:
since this is _almost_ Python, a `.gitattributes` to mark it as `linguist-language=python` might be nice.

Reply:
I was thinking about how best to have my editor automatically recognize this file as python... do you know if vscode uses `.gitattributes` to determine the language for syntax highlighting or is it only the file extension?

Reply:
oh, guess I never responded to this. `"files.associations"` is the vscode equivalent, in case this is still relevant